delta/plugins/blazegraph/src/test/scala/ch/epfl/bluebrain/nexus/delta/plugins/blazegraph/slowqueries/BlazegraphSlowQueryLoggerSuite.scala

```diff
@@ -8,33 +10,35 @@ import ch.epfl.bluebrain.nexus.delta.rdf.query.SparqlQuery
 import ch.epfl.bluebrain.nexus.delta.sdk.views.ViewRef
 import ch.epfl.bluebrain.nexus.delta.sourcing.model.{Identity, Label, ProjectRef}
 import ch.epfl.bluebrain.nexus.delta.sourcing.postgres.Doobie
-import ch.epfl.bluebrain.nexus.testkit.mu.bio.BioSuite
-import monix.bio.Task
+import ch.epfl.bluebrain.nexus.testkit.mu.ce.CatsEffectSuite
 import munit.AnyFixture
 
 import java.time.Instant
-import scala.concurrent.duration.DurationInt
+import scala.concurrent.duration.{DurationInt, FiniteDuration}
 
 object BlazegraphSlowQueryLoggerSuite {
   private val LongQueryThreshold                        = 100.milliseconds
   private val StoreWhichFails: BlazegraphSlowQueryStore = new BlazegraphSlowQueryStore {
-    override def save(query: BlazegraphSlowQuery): Task[Unit] =
-      Task.raiseError(new RuntimeException("error saving slow log"))
+    override def save(query: BlazegraphSlowQuery): IO[Unit] =
+      IO.raiseError(new RuntimeException("error saving slow log"))
 
-    override def removeQueriesOlderThan(instant: Instant): Task[Unit] = Task.unit
+    override def removeQueriesOlderThan(instant: Instant): IO[Unit] = IO.unit
 
-    override def listForTestingOnly(view: ViewRef): Task[List[BlazegraphSlowQuery]] = Task.pure(Nil)
+    override def listForTestingOnly(view: ViewRef): IO[List[BlazegraphSlowQuery]] = IO.pure(Nil)
   }
 
   private val view        = ViewRef(ProjectRef.unsafe("epfl", "blue-brain"), Iri.unsafe("hippocampus"))
   private val sparqlQuery = SparqlQuery("")
   private val user        = Identity.User("Ted Lasso", Label.unsafe("epfl"))
 }
 
-class BlazegraphSlowQueryLoggerSuite extends BioSuite with Doobie.Fixture with BlazegraphSlowQueryStoreFixture {
+class BlazegraphSlowQueryLoggerSuite extends CatsEffectSuite with Doobie.Fixture with BlazegraphSlowQueryStoreFixture {
 
   override def munitFixtures: Seq[AnyFixture[_]] = List(doobie, blazegraphSlowQueryStore)
 
+  val realClock: Clock[IO]            = Clock.create[IO]
+  implicit val mockedClock: Clock[IO] = ceClockMocked(_ => IO.pure(Instant.EPOCH.toEpochMilli), realClock.monotonic)
```

Comment:
I'm not sure what the purpose of this clock is? I'm not saying it's not needed, I just don't understand it. It feels like it could be a standard test fixture like `ceClock`, with a meaningful name?